sycl/test-e2e/ESIMD/performance_tests/instruction_count.py

```diff
@@ -0,0 +1,27 @@
+import os
+import re
+import sys
+
```

Comment:
nit: extra newline

Reply:
extra newline is added by the formatting tool